rkt/rkt.go

```diff
@@ -44,6 +44,7 @@ var (
 		Debug              bool
 		Help               bool
 		InsecureSkipVerify bool
+		TrustKeysFromHttps bool
```

Comment:
I wonder whether this should be a global parameter or rather `run` (and others) specific.